### ISSUE TITLE
Review icon component to include <i> element

### ISSUE THREAD

**Issue by landitus** [auth0/cosmos#314]
I've seen some issues with the Icon component when trying to center it (inside Empty State). Also, using a <i> we could add inline-block or block styles more effectively than when styling the <svg> tag

**Comment by nkohari** [comment]
Re: the use of `<Icon.Element>` vs exporting `StyledIcon`, the `Icon.Element` is already available anywhere `Icon` is. (That's actually the reason I favor that approach, rather than the additional export.)

For example, this should work:

```
import styled from 'styled-components';
import { Icon } from 'cosmos';

const StyledDiv = styled.div`
  & {Icon.Element} {
    text-align: center;
  }
`

const MyComponent = props => (
  <StyledDiv>
    <Icon type="help" />
  </StyledDiv>
)

```

**Comment by landitus** [comment]
Nice. I think that strategy is related with the styling strategy, which I still have to understand fully. At time we use the `StyledCompentName` and sometime `Component.xxx strategy`. So maybe it's about stating when to use which, or reviewing and checking we use it consistently.

**Comment by nkohari** [comment]
Yep! @siddharthkp and I discussed standardizing this at one point, but decided to punt until after the PoC. I will create an issue to track this.
